applications/asset_tracker_v2/src/ext_sensors/Kconfig

```diff
@@ -9,6 +9,13 @@ menuconfig EXTERNAL_SENSORS
 
 if EXTERNAL_SENSORS
 
+config EXTERNAL_SENSORS_ACTIVITY_DETECTION_AUTO
+	bool "Set accelerometer trigger handler on initialization"
+	help
+	  Enable this option to start activity detection when the library is initialized.
+	  If the option is disabled, the accelerometer trigger handler must manually be enabled to
+	  detect activity.
```

Comment:
How about this?

```suggestion
config EXTERNAL_SENSORS_ACTIVITY_DETECTION_AUTO
	bool "Enable activity detection using the built-in accelerometer"
	help
	  If enabled, activity in passive mode is detected using the built-in accelerometer.
	  If disabled, the application will not use the built-in accelerometer and only wake up based on the movement timeout.
```

Reply:
Should default to `true`.

Reply:
The ```ext_sensors``` is a sub-module that does not have the concept of active/passive.

Reply:
And it should be set to ```false``` by default. At initialization of this library we don't know the device mode. When the device mode is read from flash and distributed to the sensor module we know if we need to set activity detection or not. This is done via the ```ext_sensors_accelerometer_trigger_callback_set(true/false)``` API.